### PR TITLE
Create new issue template

### PR DESCRIPTION
**Pull request by lwtchu**

### CODE REVIEW

**Review by Stackblocks — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/NEW_ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+assignees: ''
+---
+
+<!-- Please add the book labels and the title of the book in the title of this issue.  -->
+
+# New specifications or link to documentation cards
+<!-- Please give an overview of what you'd like implemented, and include screenshots if available -->
+
+# Acceptance Criteria
+<!-- How many books does this impact? Is this across all themes? Is this book specific? -->
```
Comment (kswanson33) [inline .github/ISSUE_TEMPLATE/NEW_ISSUE_TEMPLATE:14]: for AC description can we use some of the points we discussed at last retro about what is AC?

maybe there could be another section just for impact?

**Review by kswanson33 — COMMENTED** [review]
there seem to be 2 sections to the template: feature stuff (specs & AC) & directions on how to get test content (sample markup & data on how to download the book). idk what this means i'm mostly just observing. i guess maybe we could collect these sections under big headers & make them subheaders?

a few other things i can think of that may be useful:
- reminder to use labels; a few common label categories + what they mean
- section for description of impact level
- section for context info/why
- ...

File: .github/ISSUE_TEMPLATE/NEW_ISSUE_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+name: New Issue Template
+about: Please use this template for any and all new issues.
+title: ''
+assignees: ''
+---
+
+<!-- Please add the book labels and the title of the book in the title of this issue.  -->
```
[the remark this answers is not in the document]
Comment (lwtchu) [reply .github/ISSUE_TEMPLATE/NEW_ISSUE_TEMPLATE:8]: oh let me rephrase..it's on the old one too
